src/Elmish.WPF/Logging.fs

```diff
@@ -0,0 +1,130 @@
+﻿module Elmish.WPF.Logging
```

Comment:
As long as everything is internal, you could just write `module internal ...` and remove `internal` from everything here.

Then again, it might perhaps be better to have individual control of each type/function for now?

However, I'm unsure if the IDE will show an empty module if it's public but contains no public members. So perhaps just add `internal` to the module and we'll remove it once we start making thing public.